src/Azs.Compute.Admin/custom/New-AzsComputeQuota.ps1

```diff
@@ -6,9 +6,9 @@ Creates or Updates a Compute Quota with the provided quota parameters.
 .Example
 To view examples, please use the -Online parameter with Get-Help or navigate to: https://docs.microsoft.com/en-us/powershell/module/azs.compute.admin/new-azscomputequota
 .Inputs
-Microsoft.Azure.PowerShell.Cmdlets.ComputeAdmin.Models.Api20180209.IQuota
+Microsoft.Azure.PowerShell.Cmdlets.ComputeAdmin.Models.Api202101.IQuota
 .Outputs
-Microsoft.Azure.PowerShell.Cmdlets.ComputeAdmin.Models.Api20180209.IQuota
+Microsoft.Azure.PowerShell.Cmdlets.ComputeAdmin.Models.Api202101.IQuota
```

Comment:
Did we add a new property in Quota in the api version 20210101 ?

Reply:
We added new properties to the model. On Create these properties are optional. No additional parameters for the cmdlets.